src/include/test/unit_ivf_flat_group.cc

```diff
@@ -160,6 +158,7 @@ TEST_CASE("ivf_flat_group: write constructor - create", "[ivf_flat_group]") {
 
 TEST_CASE(
     "ivf_flat_group: write constructor - create and open", "[ivf_flat_group]") {
+  bool debug = false;
```

Comment:
(Minor) It looks like this is unused, can we removed?

Reply:
Hmm.  We may be out of sync.  All of the declared `debug` are used in the current version of the branch.

Reply:
Can you share what code you are looking at? In the code in this PR I don't see any references to `debug` in this `TEST_CASE`:

<img width="770" alt="Screenshot 2024-02-09 at 11 22 06 AM" src="https://github.com/TileDB-Inc/TileDB-Vector-Search/assets/1396242/d1dc5bac-13f6-404b-8596-3357b28b42ec">